### ISSUE TITLE
Possible Inconsistency between the Description and the Code?

### ISSUE THREAD

**Issue by gaoyuankidult** [openai/robosumo#3]
https://github.com/openai/robosumo/blob/1862397b43990b663ea236cd77a56d0dd41c9f09/robosumo/envs/agents.py#L210

Hi, 

When I read the paper, I found following text describing the observation of the agent to be

> During the game, each agent observes positions of itself and the opponent, its
own joint angles, the corresponding velocities, and the forces exerted on its own body (i.e., equivalent
of tactile senses).  (Paper) 4.2 Competitive (first paragraph)

However in the code, agent.py. get_obs function lines 190-212 indicates that the forces of the opponents are also included. Could it be that I read it wrong or some mistake?

Thanks a lot for your time and consideration on this matter!

Best,

**Comment by alshedivat** [comment]
Good catch. The code is correct -- the agent additionally observed forces exerted on the opponent's torso (not on any other limbs).

**Comment by SigmaBM** [comment]
Hi, @alshedivat I recorded the observations at every step, but found that observed forces exerted on agent itself and its opponent are always zero.
